common/web/types/src/ldml-keyboard/ldml-keyboard-xml-reader.ts

```diff
@@ -12,7 +12,7 @@ export default class LDMLKeyboardXMLSourceFileReader {
   private boxArrays(source: any) {
     boxXmlArray(source?.keyboard, 'layers');
     boxXmlArray(source?.keyboard?.displays, 'display');
-    boxXmlArray(source?.keyboard?.displays, 'displayOptions');
+    // boxXmlArray(source?.keyboard?.displays, 'displayOptions');
```

Comment:
I'm not sure why this is being commented out? Is it not required because it's always a single entry?

Reply:
It's always a single entry. Should be deleted.

Reply:
done